core/rio/api/src/main/java/org/eclipse/rdf4j/rio/helpers/SimpleParseLocationListener.java

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (c) 2015 Eclipse RDF4J contributors.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Distribution License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+package org.eclipse.rdf4j.rio.helpers;
+
+import org.eclipse.rdf4j.rio.ParseLocationListener;
+
+/**
+ * A simple implementation of {@link ParseLocationListener}.
+ * 
+ * @author Peter Ansell
+ */
+public class SimpleParseLocationListener implements ParseLocationListener {
```

Comment:
To what purpose would you expect to put this implementation, other than unit testing?